runtime/doc/options.txt

```diff
@@ -1621,15 +1621,6 @@ A jump table for the options with a short description can be found at |Q_op|.
 		J	A |sentence| has to be followed by two spaces after
```

Comment:
BTW, `B` flag you mentioned in comment refers to the `<` flag, this reference should be removed as well. (And no, I do not remove it.)